llvm/lib/Target/AArch64/AArch64ISelLowering.cpp

```diff
@@ -1345,18 +1350,30 @@ AArch64TargetLowering::AArch64TargetLowering(const TargetMachine &TM,
     }
 
     // AArch64 has implementations of a lot of rounding-like FP operations.
+    // And the same for FMAXNUM_IEEE and FMINNUM_IEEE.
```

Comment:
Comment doesn't add anything

Reply:
Yes, but that doesn't need clarification. The existing comment isn't really helpful either

Reply:
OK. Let's remove them.